bundler/spec/commands/binstubs_spec.rb

```diff
@@ -289,7 +285,7 @@
 
         bundle "binstubs rack --shebang jruby"
 
-        expect(File.open(bundled_app("bin/rackup")).gets).to eq("#!/usr/bin/env jruby\n")
+        expect(File.read(bundled_app("bin/rackup"), 64)).to start_with("#!/usr/bin/env jruby\n")
```

Comment:
Can you use `File.readlines(...).first' instead? That "64" looks ugly and brittle.

Reply:
I can, but I'd prefer not.  The main reason I used it is that the whole file doesn't need to be read, which `File.readlines` does.  One could use intermediate values, or lose a set of paren's, but that's not allowed.

I'll have a look at the other test files.

Reply:
I know, but this file has like 30 lines, it's not going to make any difference. Reading it entirely is fine. Please change it.

Reply:
Done.